lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -0,0 +1,132 @@
+{{#if this.registration.license}}
+    <OsfButton
+        data-analytics-name='View license'
+        local-class='LinkButton'
+        @type='link'
+        @onClick={{action (mut this.shouldShowLicense) true}}
+    >
+        {{this.registration.license.name}}
+    </OsfButton>
+
+    {{#if this.shouldShowLicense}}
+        <BsModal
+            @onHide={{action (mut this.shouldShowLicense) false}}
+            as |modal|
+        >
+            <modal.header>
+                <h4 class='modal-title'>
+                    {{t 'registries.registration_metadata.license'}}
+                </h4>
+            </modal.header>
+            <modal.body>
+                {{#if this.registration.license.url}}
+                    <p>
+                        <OsfLink
+                            data-analytics-name='License URL'
+                            @href={{this.registration.license.url}}
+                        >
+                            {{this.registration.license.name}}
+                        </OsfLink>
+                    </p>
+                {{/if}}
+                <LicenseText @node={{this.registration}} />
+            </modal.body>
+        </BsModal>
+    {{/if}}
+{{else}}
+    {{t 'registries.registration_metadata.no_license'}}
+{{/if}}
+
+{{#if this.editMode}}
+    <BsModal @onHide={{action this.onHideModal}} as |modal|>
+        <modal.header>
+            <h4 class='modal-title'>
+                {{t 'registries.registration_metadata.select_license'}}
+            </h4>
+        </modal.header>
+        <modal.body>
+            <ValidatedModelForm
+              @model={{this.registration}}
+              @onSave={{action this.onSave}}
+              @onError={{action this.onError}}
+              as |form|>
+                <form.select
+                    @valuePath='license'
+                    @search={{perform this.queryLicenses}}
+                    @selected={{this.selectedLicense}}
+                    @options={{this.licensesAcceptable}}
+                    @onchange={{action this.changeLicense}}
+                    @noMatchesMessage={{t 'registries.registration_metadata.no_matches'}}
+                    @placeholder={{t 'registries.registration_metadata.add_license'}}
+                    @renderInPlace={{true}}
+                    as |license|
+                >
+                    {{license.name}}
+                </form.select>
+
+                <div local-class='small help-link'>
+                    <OsfLink
+                        @target='_blank'
+                        @rel='noopener'
+                        @href={{this.helpLink}}
+                    >
+                        {{t 'app_components.license_picker.faq'}}
+                    </OsfLink>
+                </div>
+
+                {{#if this.selectedLicense}}
+                    {{#if this.requiredFields}}
+                        <br>
```

Comment:
We generally shouldn't use `<br>` for formatting. This looks like something better served by styling on the custom input below?